src/services/configService.ts

```diff
@@ -178,28 +190,24 @@ export class ConfigService {
         || tenantId,
       appId: this.getOption(constants.variableKeys.appId)
         || process.env.AZURE_CLIENT_ID
-        || appId
+        || appId,
     }
     config.provider.resourceGroup = (
       this.getOption("resourceGroup", config.provider.resourceGroup)
     ) || AzureNamingService.getResourceName(options);
 
-    // Get property from `runFromBlobUrl` for backwards compatability
```

Comment:
Is this no longer required for backwards compatibility?

Reply:
No, the tool is going to take the opinion that windows should run from package and Linux can only be external. It's not going to break anything, since Linux hasn't existed yet and running Windows from external package causes significant cold start and is not recommended anyway.